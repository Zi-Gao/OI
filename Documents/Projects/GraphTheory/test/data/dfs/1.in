106 490
61 63 16832
59 19 2159
96 1 2626
21 88 5477
39 101 21265
40 6 9478
14 104 19319
88 3 825
35 60 16866
74 44 22739
89 68 30518
79 90 17635
62 99 4236
47 3 23458
81 30 23588
84 68 7081
23 77 18810
36 96 23098
79 89 7606
61 27 19904
21 13 25846
3 16 18901
24 102 13854
70 46 615
77 53 25141
58 66 21468
47 77 27000
55 11 11330
79 85 24485
29 75 21991
66 87 5685
105 73 7672
100 37 15990
103 34 12787
12 38 3782
103 47 9487
57 47 7526
1 86 27365
17 2 26765
11 75 18182
27 64 14032
67 95 18751
51 71 27648
58 75 31923
55 6 28151
60 59 1056
15 73 4638
89 76 2015
13 65 6031
74 42 10881
57 14 15126
12 5 16159
76 91 18828
39 46 18606
89 97 2613
99 90 27637
62 84 15918
27 43 1538
83 82 4880
96 77 5926
32 78 5708
76 90 14054
29 28 17395
71 83 24413
18 56 1760
25 92 7947
74 49 14728
77 68 13825
57 67 15827
85 29 24550
44 96 26571
48 100 7763
15 47 2570
2 57 20332
11 29 17453
68 58 19146
34 40 4785
45 44 13070
90 18 23814
14 32 13490
50 10 10693
22 7 6903
61 92 2624
20 82 2563
98 4 1511
48 105 22916
71 38 9025
76 25 3454
65 23 28511
81 13 21252
86 51 26136
45 5 17678
54 27 11375
7 58 8625
64 40 31723
96 48 4701
66 97 14553
3 44 18925
11 50 18080
100 77 4532
16 33 5298
24 7 31425
81 6 25546
39 11 23319
51 30 13928
50 15 5778
85 62 4309
104 52 416
89 19 27665
49 29 37
35 7 23669
66 26 27931
16 75 14326
73 98 26580
55 2 20811
18 2 27107
96 14 29791
89 96 5393
12 61 1497
56 30 2811
74 102 19104
60 103 26652
88 86 15783
89 51 14960
87 54 18787
78 98 4096
66 51 1974
41 27 5101
64 2 9784
99 52 12499
25 47 1572
101 88 26015
98 1 30297
42 30 30254
74 50 19233
8 44 29515
77 15 27330
53 34 19945
22 42 30481
35 14 22444
96 26 23408
86 66 18203
8 40 1025
14 60 8210
65 2 11768
35 80 20817
83 40 27215
22 37 32311
50 66 25420
59 88 22540
7 79 2709
45 51 11364
62 90 22729
4 16 30337
96 54 2262
78 3 24516
29 14 2179
76 91 16268
4 60 1259
92 26 25864
96 6 21716
15 23 22004
99 89 23598
89 71 17211
8 79 11042
34 19 20709
86 19 24680
72 68 230
96 68 2543
22 61 10409
77 41 12280
28 9 26471
94 42 13863
40 71 10153
95 7 6981
102 88 24718
20 5 11173
37 17 7076
59 53 17253
37 85 19065
41 47 5382
50 14 17198
27 49 24077
40 26 23984
80 38 22193
28 25 13162
97 15 19131
71 72 387
63 83 31014
89 91 12811
9 28 25890
66 52 10915
31 4 32390
78 70 16264
75 29 16598
43 82 8191
56 77 28515
88 46 19870
8 78 14904
96 65 16865
96 24 10953
72 96 24075
65 102 2210
94 62 12105
5 90 17804
95 45 1416
87 71 15830
47 28 25991
100 73 1235
74 53 3846
68 71 24228
27 49 13854
103 68 16356
27 102 15149
23 30 218
9 43 11992
50 82 13617
11 80 22545
51 88 6768
69 1 20517
50 72 7843
67 28 11574
13 4 28148
89 69 11705
9 72 4390
48 49 30663
91 79 16434
28 60 21861
28 54 29556
41 63 29457
34 67 17191
35 59 6196
37 12 6182
53 27 10565
99 13 25128
36 54 4552
22 94 18167
82 65 2158
23 42 23374
49 5 2551
61 55 16730
96 91 30640
83 1 21028
31 96 3233
33 69 3647
15 35 26818
27 54 18323
91 98 21459
71 34 6433
13 10 8621
101 76 8973
55 35 24609
20 89 28755
47 39 15046
5 11 14584
24 32 23360
50 54 11916
83 104 3305
40 33 31324
68 9 8418
59 65 23471
22 2 3062
26 103 14800
25 74 2518
53 15 20703
27 29 18833
31 76 12705
37 85 21855
24 47 27586
21 76 7189
58 59 14148
89 4 19667
79 98 7444
50 100 9688
38 24 18430
77 66 32120
41 93 7791
29 79 14501
23 11 23248
91 30 9370
16 67 23385
65 13 20854
31 31 9128
2 88 10736
84 14 22926
23 87 10353
64 45 13254
48 54 17456
50 95 11407
8 57 4390
102 66 9067
61 104 9200
81 26 16491
25 73 9990
28 67 12488
72 1 16576
61 21 1995
3 31 549
14 50 37
40 78 9348
48 5 22851
3 101 28723
84 78 26960
6 79 20758
71 13 2008
50 48 28088
53 97 32328
47 73 7195
91 52 716
63 84 23439
104 101 14122
7 66 7461
38 48 2225
66 69 11428
45 26 28507
43 38 2570
18 105 14764
81 35 2033
3 73 8182
86 35 7555
59 68 5384
30 9 6838
29 5 888
4 4 25607
100 48 11775
23 39 5879
41 21 3753
35 3 2793
47 52 9156
85 52 6715
71 73 27341
74 80 3504
6 25 10452
105 37 30850
88 69 23819
20 57 23990
43 98 9597
84 66 10118
9 39 18205
20 25 24579
102 18 16814
73 58 17974
102 46 11576
18 14 23915
49 98 25088
9 21 31855
1 81 27965
91 59 16613
87 78 25605
76 18 12029
75 41 4897
53 42 16430
37 66 9680
15 16 17361
59 88 30362
50 47 19102
83 92 9412
68 72 29315
54 22 3594
94 22 23114
58 44 16288
33 75 23817
103 51 8880
30 75 858
5 14 26571
55 53 27833
10 70 21004
34 52 23979
79 105 19932
14 60 12156
78 72 2194
45 78 9384
94 70 21868
89 1 12628
39 95 24978
47 54 10596
46 22 23583
60 8 3600
24 90 6247
34 79 29436
8 48 8176
39 61 14156
46 71 30430
32 25 10539
90 31 20669
34 41 10927
83 87 27881
23 17 9518
93 102 6418
85 89 5216
32 69 15748
48 69 17626
44 86 10809
5 71 6601
77 75 27878
101 45 23547
11 56 6016
85 8 18575
3 82 23531
20 54 24558
22 68 3277
88 8 26065
102 64 15553
20 77 10625
64 24 11360
27 75 6559
34 35 19665
11 39 26890
13 10 8167
33 24 12989
25 18 32752
30 10 10437
11 91 10438
76 24 32408
1 53 2246
88 6 31090
14 101 15271
97 97 29064
84 79 27261
32 21 9519
40 101 29505
78 101 6984
14 58 32686
25 41 1170
4 39 26213
15 69 9138
25 1 8839
14 40 21002
74 31 11597
74 7 19390
59 87 25153
21 18 6007
10 2 8743
16 82 11965
93 70 29121
64 45 4615
70 102 10464
77 11 16337
53 52 15244
15 85 10322
93 20 2605
55 38 16303
97 26 18160
29 47 20153
56 47 11744
57 99 9880
74 69 23152
42 84 30203
75 61 16488
2 65 16021
5 6 1903
67 103 24600
41 94 4008
103 76 27121
104 45 2848
95 98 30570
31 61 11004
60 94 20988
96 74 12460
15 81 1024
103 68 4745
96 70 31229
58 41 8357
65 73 23357
55 4 452
85 104 29303
35 23 18858
21 71 26033
53 84 8517
100 49 10400
42 101 20059
14 1 5943
6 83 11212
94 12 3615
83 39 9343
32 42 13647
15 66 21139
56 96 5380
93 59 20852
12 24 1981
10 95 27862
42 28 15372
97 92 28630
73 90 20321
60 52 16032
16 82 10005
103 80 27250
3 74 8025
81 58 25304
7 21 32735
