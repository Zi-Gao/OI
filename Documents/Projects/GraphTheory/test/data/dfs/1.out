1 53 15 16 33 24 7 21 13 4 39 11 29 5 6 25 18 2 57 14 32 42 28 9 43 38 48 49 98 54 22 37 12 61 27 64 40 26 103 34 19 35 3 31 76 90 91 30 10 70 46 71 72 68 58 41 47 52 73 77 66 51 88 8 44 86 96 65 23 17 87 78 101 45 102 74 50 82 95 100 69 80 79 85 62 84 99 89 97 92 104 105 75 63 83 93 20 59 94 60 67 55 56 81 